### PR TITLE
Update `o!tFAC #1` contest article

### PR DESCRIPTION
**Pull request by Hiviexd**
will do one last update later when the badge request is concluded

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Contests/o!tFAC/1/en.md [hunk 1]
```diff
@@ -71,6 +83,9 @@ The osu!taiko Featured Artist Cup #1 is run by various community members.
 - **Creativity (10 Points):** Points will be assigned based on how the submission uses creative and fitting elements to make itself stand out.
 
 [flag_AR]: /wiki/shared/flag/AR.gif "Argentina"
+[flag_KR]: /wiki/shared/flag/KR.gif "South Korea"
+[flag_IT]: /wiki/shared/flag/IT.gif "Italy"
```
Comment (Walavouchey) [inline wiki/Contests/o!tFAC/1/en.md:87]:
```suggestion
[flag_IT]: /wiki/shared/flag/IT.gif "Italy"
[flag_KR]: /wiki/shared/flag/KR.gif "South Korea"
```

File: wiki/Contests/o!tFAC/1/en.md [hunk 2]
```diff
@@ -45,12 +45,24 @@ The osu!taiko Featured Artist Cup #1 is run by various community members.
...
 
-¹: ![][flag_TN] [Hivie](https://osu.ppy.sh/users/14102976) and ![][flag_US] [Nifty](https://osu.ppy.sh/users/4956097) will mod and push the winning entry to ranked if the winner chooses to, and **if they get the necessary difficulties to form a rankable spread if needed**.
+¹: ![][flag_TN] [Hivie](https://osu.ppy.sh/users/14102976) and ![][flag_US] [Nifty](https://osu.ppy.sh/users/4956097) will mod and push the winning entry to Ranked if the winner chooses to, and **if they get the necessary difficulties to form a rankable spread if needed**.
 
 ## Links
 
 - [Discussion thread](https://osu.ppy.sh/community/forums/topics/1520800)
 - [Discord server](https://discord.gg/jMsVqwPVrD)
+- [Results sheet](https://docs.google.com/spreadsheets/d/1v8B409syFzzAfgoxXKx6ptitrtNPYDa09Avkt8PeXLI/edit?usp=sharing)
+- [Results stream VOD](https://www.twitch.tv/videos/1434459833)
```
Comment (Walavouchey) [inline wiki/Contests/o!tFAC/1/en.md:55]:
```suggestion
- [Results announcement VOD](https://www.twitch.tv/videos/1434459833)
```

**Review by Walavouchey — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Comment by Walavouchey** [comment]
try to not pr from master next time ty

**Comment by Hiviexd** [comment]
oops sry

**Pull request merged**